src/server/GA_parameters.py

```diff
@@ -0,0 +1,25 @@
+class GA_parameters:
+    def __init__(self, 
+        nb_epochs: int,
+        pop_size: int,
+        crossover_rate: float,
+        mutation_rate: float) :
+        self.__nbEpochs = nb_epochs
+        self.__popSize = pop_size
+        self.__crossoverRate = crossover_rate
+        self.__mutationRate = mutation_rate
+
+    def get_parameters(self):
```

Comment:
This method is duplicated with the DRL parameter class. What is the purpose of using classes here ? If using a class is motivated, consider using polymorphism